pages/common/gh-issue.md

```diff
@@ -5,28 +5,28 @@
 
 - Display a specific issue:
 
-`gh issue view {{issue_number}}`
+`gh issue view {{1..infinity}}`
```

Comment:
```suggestion
`gh issue view {{issue_number}}`
```
Only 1 value is supported here
```bash
$ gh issue view 2068 9557
accepts 1 arg(s), received 2
```

Reply:
Yes, you misinterpreted [range syntax](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#range-syntax): it is not an ellipsis, it's always a single argument.

Reply:
oh! you are specifying the range of issues in GH. No, this is not cool. It's similar to put an e-mail Regex for `email` values in examples. It isn't clear.